cmd/gateway_validate.go

```diff
@@ -210,6 +212,26 @@ this command unless --online flag is used.
 			if len(validateCmdKongStateFile) == 0 {
 				validateCmdKongStateFile = []string{"-"}
 			}
+
+			// Iterate over the input values and validate them against the keys in entityMap
+			for _, value := range validateCmdOnlineEntitiesFilter {
+				// Check if the value is valid by comparing it with keys in EntityMap
+				if _, exists := validate.EntityMap[value]; !exists {
+					// Generate an error message with the list of valid keys
+					listOfKeys := make([]string, 0, len(validate.EntityMap))
+					for key := range validate.EntityMap {
```

Comment:
Yes I think we need this format for visibilityto have only the entties listed at the top. Also no issue with iteration and sorting as it will do the "return" at the first invalid iteration so it will only run once at max :)